pkg/deploy/dev-workspace/dev_workspace.go

```diff
@@ -113,6 +116,12 @@ func ReconcileDevWorkspace(deployContext *deploy.DeployContext) (bool, error) {
 		return true, nil
 	}
 
+	// Check if exists devworkspace operator csv is already installed
+	devWorkspaceOperatorCSVExists := isDevWorkspaceControllerCSVExists(deployContext)
+	if devWorkspaceOperatorCSVExists {
```

Comment:
As far as I understand it means that if you install Che with chectl operator installer, DWO will be installed from resources but not with OLM dependency, which may be unexpected in some corner cases, like when WTO is installed after Che with chectl operator installer.

So, I would say Che on OpenShift should install DWO with olm, or at least into openshift-operators namespace...

Reply:
Yes, make sense your comments. I think we need to have a dedicate issue for this.